atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -110,6 +111,7 @@ class OrchestrateBuildPlugin(BuildStepPlugin):
     """
 
     EXCLUDE_FILENAME = 'exclude-platform'
+    UNREACHABLE_CLUSTER_LOAD = -1
```

Comment:
Right, but at least it would be numerical. I'm thinking I should rewrite this and change `get_enabled_clusters_for_platform` instead